### DISCUSSION TITLE
GSoD 2023

### DISCUSSION THREAD

**Discussion by mattytrentini** [micropython/micropython#11330] [Announcements and news]
I'm delighted to officially announce that MicroPython was selected to be part of [Google's Season of Docs](https://developers.google.com/season-of-docs) for 2023! GSoD provides support for open-source projects to improve their documentation by connecting and funding Technical Writers for the selected projects.

We are amongst thirteen [organizations selected for GSoD](https://developers.google.com/season-of-docs/docs/participants) and the [MicroPython proposal](https://docs.google.com/document/d/1JtwBneelTCVpFIw4ER0POs_-jksD96I00qqanieELMQ) is ambitious, tackling API documentation using inline annotations and stubfiles, extracting generic from platform-specific documentation, restructuring the documentation hierarchy...and more.

We have received _many_ excellent applications from Technical Writers but I'm excited to also announce that **Jim Mussared** applied, and has been selected, for the GSoD Technical Writer role in 2023!

Jim will be well-known to many in the community but, for those that don't know @jimmo, he's a long-time core contributor to MicroPython for both software and documentation development. His experience was impossible to overlook and we can't wait to see his efforts to improve MicroPython's documentation!

**Comment by peterhinch** [comment]
What was the outcome of this?
